modules/dnn/test/test_darknet_importer.cpp

```diff
@@ -453,15 +463,24 @@ TEST_P(Test_Darknet_nets, YOLOv3)
         applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_MYRIAD, CV_TEST_TAG_DNN_SKIP_IE_NGRAPH);
 
     // batchId, classId, confidence, left, top, right, bottom
-    Mat ref = (Mat_<float>(9, 7) << 0, 7,  0.952983f, 0.614622f, 0.150257f, 0.901369f, 0.289251f,  // a truck
-                                    0, 1,  0.987908f, 0.150913f, 0.221933f, 0.742255f, 0.74626f,   // a bicycle
-                                    0, 16, 0.998836f, 0.160024f, 0.389964f, 0.417885f, 0.943716f,  // a dog (COCO)
-                                    1, 9,  0.384801f, 0.659824f, 0.372389f, 0.673926f, 0.429412f,  // a traffic light
-                                    1, 9,  0.733283f, 0.376029f, 0.315694f, 0.401776f, 0.395165f,  // a traffic light
-                                    1, 9,  0.785352f, 0.665503f, 0.373543f, 0.688893f, 0.439245f,  // a traffic light
-                                    1, 0,  0.980052f, 0.195856f, 0.378454f, 0.258626f, 0.629258f,  // a person
-                                    1, 2,  0.989633f, 0.450719f, 0.463353f, 0.496305f, 0.522258f,  // a car
-                                    1, 2,  0.997412f, 0.647584f, 0.459939f, 0.821038f, 0.663947f); // a car
+    const int N0 = 3;
+    const int N1 = 6;
+    static const float ref_[/* (N0 + N1) * 7 */] = {
+//0, 16, 0.998836f, 0.160024f, 0.389964f, 0.417885f, 0.943716f,
+//0, 1, 0.987908f, 0.150913f, 0.221933f, 0.742255f, 0.746261f,
+//0, 7, 0.952983f, 0.614621f, 0.150257f, 0.901368f, 0.289251f,
```

Comment:
Why do we need these commented lines?

Reply:
They are dumped after NMS through code under `#if 0`
But test itself fails with them for some reason.

Reply:
But there are 3 lines with the same values below

Reply:
You are right! They are duplicates from batch size 1/2.
Fixed.